xml_inscriptions/transcribed/MI.AA.UM.KM.G.21182.xml

```diff
@@ -55,9 +55,14 @@
    <!--facsimile element goes here--><text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+        <ab>
+           <lb n="1"/> <persName><name key="Νεμαισᾶς">Νεμαισᾶς</name></persName> ξʹ, <persName><name key="Ἀπολλωνάριν">Ἀπολλωνάριν</name></persName>  λεʹ, <persName><name key="Ἡφαιστᾶς">Ἡφαιστᾶς</name></persName>  <hi rend="supraline">μ</hi>, <persName><name key="Νεμεσάμμων>Νεμεσάμμω<supplied reason="lost">ν</supplied></name></persName>. κ Ἁθὺρ <hi rend="supraline">ια</hi>..
```

Comment:
I don't have access to the bibliography, but is Ἀπολλωνάριν accusative?

Reply:
I think so, but I checked the Greek inscriptions online and I couldn't find this name attested in another form, like in the nominative... so I just left it as is.

Reply:
I'm not sure of the context but yes, Ἀπολλωνάριν should be accusative (of the more normal Apollinaris)

Reply:
OK, let's index under Ἀπολλωνάρις then.